services/frontend-service/src/ui/App/index.tsx

```diff
@@ -88,13 +88,12 @@ export const App: React.FC = () => {
         }
     );
 
-    const { app, version } = useReleaseDialog(({ app, version }) => ({ app, version }));
-    const releaseInfo = useReleaseInfo(app, version);
+    const { app, version } = useReleaseDialogParams();
```

Comment:
please rename this function in the commit message as well.

Reply:
oh yes. right :+1: done